test/src/test/testtool/Receivers/NonTransacted/NoInProcess/scenario01.properties

```diff
@@ -10,11 +10,11 @@ step2.manage.db.read  = ../../common/md-delete-result.xml
 step3.manage.db.write = common/md-insert.xml
 step4.manage.db.read  = ../../common/md-insert-result.xml
 
-step5.java.callback.read = ignore
-step6.java.callback.writeline = <root forward="ready"/>
+step5.javaCallback.read = ignore
+step6.javaCallback.writeline = <root forward="ready"/>
 
-step7.waitfor.db.write = common/md-wait-till-message-done.xml
-step8.waitfor.db.read = ignore
+step7.wait.for.db.write = common/md-wait-till-message-done.xml
```

Comment:
Is dit het scenario wat misging? In het ticket gaat het om .classname en .service als suffix, ik weet niet of dat nog verschil maakt?

Reply:
Ik had erbij moeten zetten in het PR dat scenarios zijn aangepast als "real life test" dat de aanpassing werkt. Dit is wel iets wat mis ging toen ik het scenario wou uitbreiden een paar weken geleden, daarom dat ik hierop teruggreep bij dit ticket.